### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by Blake-Madden**
Note that I changed a `memcopy` to `memcpy`. I assume you were referring to the C `memcpy` function.

### CODE REVIEW

File: wdk-ddi-src/content/d3d12umddi/ns-d3d12umddi-d3d12ddi_video_encoder_resolution_support_limits_0080_2.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Indicates the maximum number that can be used in [**D3D12DDI_VIDEO_ENCODER_INTRA
 
 ### -field SubregionBlockPixelsSize
 
-Indicates, for the associated resolution, the size in pixels of the squared regions that will be used to partition the frame for the subregion layout (slices) semantics. The resolution of the frame will be rounded up to be aligned to this value when the rame is partitioned in bloks. The configuration of the suregion partitioning will use a number of squared subregions that have their size in pixels according to the value of this member.
+Indicates, for the associated resolution, the size in pixels of the squared regions that will be used to partition the frame for the subregion layout (slices) semantics. The resolution of the frame will be rounded up to be aligned to this value when the ram is partitioned in blocks. The configuration of the subregion partitioning will use a number of squared subregions that have their size in pixels according to the value of this member.
```
Comment (tedhudek) [inline wdk-ddi-src/content/d3d12umddi/ns-d3d12umddi-d3d12ddi_video_encoder_resolution_support_limits_0080_2.md]:
```suggestion
Indicates, for the associated resolution, the size in pixels of the squared regions that will be used to partition the frame for the subregion layout (slices) semantics. The resolution of the frame will be rounded up to be aligned to this value when the frame is partitioned in blocks. The configuration of the subregion partitioning will use a number of squared subregions that have their size in pixels according to the value of this member.
```

Interesting miss here

File: wdk-ddi-src/content/ucmtypes/nf-ucmtypes-ucm_pd_request_data_object_init_ulong.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The ULONG value to set in the <b>Ul</b> member of   <a href="/windows-hardware/d
 
 ## -remarks
 
-For information about Request Data Objects, see the Power Delivery specification. There are different types of Request Data Objects and the type depends on the Power Data Object that is specified in the <b>ObjectPosition</b> member of <a href="/windows-hardware/drivers/ddi/ucmtypes/ns-ucmtypes-_ucm_pd_request_data_object">UCM_PD_REQUEST_DATA_OBJECT</a>. The  source buffer is little-endian format and the client driver can memcopy the Request Data Objects from the hardware into an array of <b>UCM_PD_REQUEST_DATA_OBJECT</b> structures.
+For information about Request Data Objects, see the Power Delivery specification. There are different types of Request Data Objects and the type depends on the Power Data Object that is specified in the <b>ObjectPosition</b> member of <a href="/windows-hardware/drivers/ddi/ucmtypes/ns-ucmtypes-_ucm_pd_request_data_object">UCM_PD_REQUEST_DATA_OBJECT</a>. The source buffer is little-endian format and the client driver can memcpy the Request Data Objects from the hardware into an array of <b>UCM_PD_REQUEST_DATA_OBJECT</b> structures.
```
Comment (tedhudek) [inline wdk-ddi-src/content/ucmtypes/nf-ucmtypes-ucm_pd_request_data_object_init_ulong.md]: @mhopkins-msft as content owner, any thoughts on this use of memcpy?

**Comment by tedhudek** [comment]
More great finds, @Blake-Madden, thanks again.  BTW if you are looking to widen the type of content your tool covers, you could try running it on the https://github.com/MicrosoftDocs/windows-driver-docs repo.  That repo contains the conceptual guidance documentation that describes how and when to use the APIs that are documented in this repo.  So instead of API ref docs, it's walkthroughs, overviews, procedures, that kind of thing.

**Comment by Blake-Madden** [comment]
@tedhudek I will indeed check that repo out; thanks for the heads up!

File: wdk-ddi-src/content/ucmtypes/nf-ucmtypes-ucm_pd_request_data_object_init_ulong.md (hunk 2 above)
Comment (mhopkins-msft) [reply wdk-ddi-src/content/ucmtypes/nf-ucmtypes-ucm_pd_request_data_object_init_ulong.md]: I'd rather say:

`The source buffer is little-endian format. The client driver can call the [memcpy](/cpp/c-runtime-library/reference/memcpy-wmemcpy) function to get the Request Data Objects from the hardware into an array of <b>UCM_PD_REQUEST_DATA_OBJECT</b> structures.`

Comment (tedhudek) [reply wdk-ddi-src/content/ucmtypes/nf-ucmtypes-ucm_pd_request_data_object_init_ulong.md]:
```suggestion
For information about Request Data Objects, see the Power Delivery specification. There are different types of Request Data Objects and the type depends on the Power Data Object that is specified in the <b>ObjectPosition</b> member of <a href="/windows-hardware/drivers/ddi/ucmtypes/ns-ucmtypes-_ucm_pd_request_data_object">UCM_PD_REQUEST_DATA_OBJECT</a>. The source buffer is little-endian format. The client driver can call the [**memcpy**](/cpp/c-runtime-library/reference/memcpy-wmemcpy) function to get the Request Data Objects from the hardware into an array of <b>UCM_PD_REQUEST_DATA_OBJECT</b> structures.

```

**Pull request merged**